src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const allProperties = sourceString.split(';');
+
+  const resultObject = allProperties.reduce((currentObject, property) => {
+    const part = property.split(':');
+    const partName = part[0]?.trim();
+    const partValue = part[1]?.trim();
+
+    if (partName && partValue) {
+      currentObject[partName] = partValue;
+    }
+
+    return currentObject;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. However, in this case, there are no chained methods, but it's important to remember this rule for future reference.